package.json

```diff
@@ -1,63 +1,64 @@
+    "name": "pastanaga",
+    "version": "0.0.0",
+    "scripts": {
+        "ng": "ng",
+        "prestart": "npm run icon_list",
+        "start": "ng serve demo",
+        "deploy": "npm run icon_list && ng deploy --base-href=/pastanaga-angular/",
+        "build:demo": "npm run icon_list && ng build demo --configuration production --base-href /pastanaga-angular/v2/ && cp ./dist/demo/v2/index.html ./dist/demo/v2/404.html",
+        "build": "cp src/pastanaga*.scss . && ng build --configuration production && cp ./README.md ./dist/pastanaga-angular && cp -r projects/pastanaga-angular/src/styles ./dist/pastanaga-angular/lib && cp -r ./projects/pastanaga-angular/src/assets ./dist/pastanaga-angular && rm pastanaga*.scss",
+        "export:css": "rm -rf ./dist/css && mkdir ./dist/css && sass ./src/standalone-css.scss ./dist/css/pastanaga.css -I ./src -s compressed && cp -r projects/pastanaga-angular/src/assets/fonts ./dist/css",
+        "watch": "ng build --watch --configuration development",
+        "test:lib": "jest --config ./projects/pastanaga-angular/jest.lib.config.js",
+        "test:lib:watch": "jest --watch --config ./projects/pastanaga-angular/jest.lib.config.js",
+        "test": "npm run test:lib",
+        "test:ci": "jest --config ./projects/pastanaga-angular/jest.lib.config.js --runInBand --ci",
+        "get_version": "cat ./projects/pastanaga-angular/package.json | grep version  | head -1 | awk -F: '{ print $2 }' | sed 's/[\",]//g' | tr -d '[[:space:]]'",
+        "icon_list": "python -c \"import json; import xml.dom.minidom; print(json.dumps([el.getAttribute('id') for el in xml.dom.minidom.parseString(open('projects/pastanaga-angular/src/assets/glyphs-sprite.svg').read()).getElementsByTagName('symbol')]))\" > projects/demo/src/assets/glyphs.json",
+        "prettier": "prettier --write ./projects/pastanaga-angular/src"
+    },
+    "private": true,
+    "dependencies": {
+        "@angular/animations": "~13.0.1",
+        "@angular/cdk": "^13.0.1",
+        "@angular/common": "~13.0.1",
+        "@angular/compiler": "~13.0.1",
+        "@angular/core": "~13.0.1",
+        "@angular/forms": "~13.0.1",
+        "@angular/platform-browser": "~13.0.1",
+        "@angular/platform-browser-dynamic": "~13.0.1",
+        "@angular/router": "^13.0.1",
+        "@briebug/jest-schematic": "^3.1.0",
+        "@ng-web-apis/common": "^1.12.1",
+        "angular-svg-icon": "^13.0.0",
+        "angular-traversal": "^1.9.6",
+        "date-fns": "^2.25.0",
+        "rxjs": "~6.6.0",
+        "tslib": "^2.3.0",
+        "zone.js": "~0.11.4"
+    },
+    "devDependencies": {
+        "@angular-builders/jest": "^13.0.0",
+        "@angular-devkit/build-angular": "~13.0.2",
+        "@angular/cli": "~13.0.2",
+        "@angular/compiler-cli": "~13.0.1",
+        "@ngneat/spectator": "^8.3.0",
+        "@types/jest": "^27.0.2",
+        "@types/node": "^12.11.1",
+        "angular-cli-ghpages": "^1.0.0-rc.2",
```

Comment:
version 1.0.0 is now out and support angular 13: https://github.com/angular-schule/angular-cli-ghpages/releases/tag/v1.0.0